Source/Scene/ImageryLayerCollection.js

```diff
@@ -326,47 +326,17 @@ ImageryLayerCollection.prototype.lowerToBottom = function (layer) {
 var applicableRectangleScratch = new Rectangle();
 
 /**
- * Asynchronously determines the imagery layer features that are intersected by a pick ray.  The intersected imagery
- * layer features are found by invoking {@link ImageryProvider#pickFeatures} for each imagery layer tile intersected
- * by the pick ray.  To compute a pick ray from a location on the screen, use {@link Camera.getPickRay}.
+ * Helper function for `pickImageryLayers` and `pickImageryLayerFeatures`.
```

Comment:
It's fine to remove the documentation above `pickImageryHelper` since it's a local function.

Also some of the doc is misformatted (I think in `@return`) which is causing CI to fail, but no need to worry about since the doc will be removed.

Reply:
@lilleyse I just removed the documentation above `pickImageryHelper`. CI now passes.

Reply:
Thanks!